### PR TITLE
DOC: Add Applications header to readme

### PR DESCRIPTION
**Pull request by adler-j**
So this is a prospective idea on how we can present the work being done with ODL to gain some two-way publicity.

Some questions that remain open:

* Is this a good format?
* Is the distinction between articles/other stuff good?
* Should we keep this in the readme or move it to some other page?

Further we need to add some more articles that we know use ODL, e.g.

* @ozanoktem and @chongchenmath and their work on shape regularization
* @MatthiasJE work on stochastic primal dual methods
* Anyone I missed?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,34 @@ Resources
 - [Code Examples](examples)
 - [API reference](https://odlgroup.github.io/odl/odl.html)
 
+Applications
+============
+ODL has been used in a wide range of research and development with several research articles and conference contributions using ODL, as well as other open source projects using ODL as a back-end. Many of these also release their code publicly can serve as examples of how to use ODL. This is a short list of such projects. If you want your project added, contact the developers.
```
Comment (kohr-h) [inline README.md]: research articles -> scientific papers
~~using ODL~~
[...] code publicly **and** can [...]
want your project added -> want to add your project to the list
contact the **maintainers or file a pull request**.

File: README.md [hunk 2]
```diff
@@ -49,6 +49,34 @@ Resources
...
+
+##### Articles
+
+
+| Description      |  Code  |
+|------------------|--------|
+| *High-level algorithm prototyping: an example extending the TVR-DART algorithm*. Conference on Discrete Geometry for Computer Imagery (DGCI), 2017 | [GitHub](https://github.com/aringh/TVR-DART) |
+| *GPUMCI, a ﬂexible platform for x-ray imaging on the GPU*. Fully3D, 2017 | N/A |
+| *Generalized Sinkhorn iterations for regularizing inverse problems using optimal mass transport*. [Arxiv](https://arxiv.org/abs/1612.02273) |  N/A |
```
Comment (kohr-h) [inline README.md]: I'd just leave the cell empty where code is not available, not write "N/A".

File: README.md [hunk 3]
```diff
@@ -49,6 +49,34 @@ Resources
...
+| *Spectral CT reconstruction with anti-correlated noise model and joint prior*. Fully3D, 2017 | [GitHub](https://github.com/adler-j/spectral_ct_examples) |
+| *Solving ill-posed inverse problems using iterative deep neural networks*. [Arxiv](https://arxiv.org/abs/1704.04058) | [GitHub](https://github.com/adler-j/learned_gradient_tomography) | 
+| *A modified fuzzy C means algorithm for shading correction in craniofacial CBCT images*. CMBEBIH, 2017 | [GitHub](https://github.com/adler-j/mfcm_article) | 
+| *Total variation regularization with variable Lebesgue prior*. [Arxiv](https://arxiv.org/abs/1702.08807) | [GitHub](https://github.com/kohr-h/variable_lp_paper) |
```
Comment (kohr-h) [inline README.md]: What's the ordering here? If it's not already this way, I suggest (at least roughly) ordering by publication date, most recent on top.

File: README.md [hunk 4]
```diff
@@ -49,6 +49,34 @@ Resources
...
+
+
+##### Other projects
+
+| Description      |  Code  |
+|------------------|--------|
+| Bindings for the EMRecon package for PET |  [GitHub](https://github.com/odlgroup/odlemrecon) |
```
Comment (kohr-h) [inline README.md]: Link to EMRecon

File: README.md [hunk 5]
```diff
@@ -49,6 +49,34 @@ Resources
...
+| ADF-STEM reconstruction using nuclear norm regularization | [GitHub](https://github.com/adler-j/odl-stem-examples) |
+| Inverse problems over Lie-groups | [GitHub](https://github.com/adler-j/lie_grp_diffeo) |
```
Comment (kohr-h) [inline README.md]: Lie groups

File: README.md (hunk 3 above)
Comment (adler-j) [inline README.md]: I agree, will get that fixed. Currently ordering is approximately by answers to the issue :O

**Comment by adler-j** [comment]
> Somewhat related: we should move the "Contributors" list to a separate file and just mention the maintainers (if at all).

I agree, it is getting clogged. That way we can also mention what they did so they get some credit for that!

**Comment by adler-j** [comment]
Fixed comments and added @chongchenmath's stuff. Merge?

**Comment by aringh** [comment]
Looks good to me.

**Pull request merged**